dockerfiles/base-ci-linux/Dockerfile

```diff
@@ -20,24 +20,32 @@ dockerfiles/rust-builder/README.md" \
 
 WORKDIR /builds
 
-# install tools and dependencies
-COPY clang9.key /etc/apt/trusted.gpg.d/debian-archive-jessie-automatic.gpg
 # config for wasm32-unknown-unknown
-COPY config /root/.cargo/config
+COPY utility/base-ci-linux-config /root/.cargo/config
+
 # gcc and clang are necessary to be able to compile crates that have build scripts
 # Most notably, clang 9 has WASM support enabled by default.
+COPY utility/debian-llvm-clang9.key /etc/apt/trusted.gpg.d/debian-archive-llvm.gpg
+
+# copy cmake
+COPY utility/cmake.sh cmake.sh
```

Comment:
that's a 38Mb binary-containing script in the repo?

Reply:
yep, looks disgusting

Reply:
But is it worse than downloading the said script over the web every time (both buildtime-wise and security-wise)?

Reply:
the file indicates in no way which version it contains. one has to open a 38Mb file and look inside (or run it), that makes it hard to track which version it is and updating potenital security fixes will most likely not happen. attaching to some kind of upstream update mechanism at least for the same minor release would be better imo.

Reply:
and for the binary blob - that is way too big for a source repository. just imagine if one would update the file more than once - the repository will grow huge. a better approach would be to use a prebuild version uploaded into say a s3 bucket (or even use a gitlab ci for that) and revert the whole repo to get it back small and handy again.